ntp-udp/src/socket.rs

```diff
@@ -352,26 +354,29 @@ fn fetch_send_timestamp_help(
                 // the timestamping does not set a message; if there is a message, that means
                 // something else is wrong, and we want to know about it.
                 if error.ee_errno as libc::c_int != libc::ENOMSG {
-                    warn!(
+                    tracing::warn!(
                         expected_counter,
-                        error.ee_data, "error message on the MSG_ERRQUEUE"
+                        error.ee_data,
+                        "error message on the MSG_ERRQUEUE"
                     );
                 }
 
                 // Check that this message belongs to the send we are interested in
                 if error.ee_data != expected_counter {
-                    warn!(
+                    tracing::info!(
                         error.ee_data,
-                        expected_counter, "Timestamp for unrelated packet"
+                        expected_counter,
+                        "Timestamp for unrelated packet"
                     );
                     return Ok(None);
                 }
             }
 
             ControlMessage::Other(msg) => {
-                warn!(
+                tracing::info!(
                     msg.cmsg_level,
-                    msg.cmsg_type, "unexpected message on the MSG_ERRQUEUE",
+                    msg.cmsg_type,
+                    "unexpected message on the MSG_ERRQUEUE",
```

Comment:
This feels like it is either a bug (and should thus be warn) or some os oddity that we just want to log as debug.